crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1856,20 +1856,26 @@ async fn get_network_metrics_cached(
 #[async_trait]
 impl ObjectProvider for PgIndexerStore {
     type Error = IndexerError;
-    async fn get_object(
+    fn get_object(
         &self,
-        id: &ObjectID,
-        version: &SequenceNumber,
+        _id: &ObjectID,
+        _version: &SequenceNumber,
     ) -> Result<sui_types::object::Object, Self::Error> {
-        self.get_sui_types_object(id, version).await
+        Err(IndexerError::NotSupportedError(
+            "Object cache is not supported".to_string(),
+        ))
+        // self.get_sui_types_object(id, version)
```

Comment:
@patrickkuo @gegaowp this will only be called when we turn on the indexer native implementation, right?